app/src/main/java/de/westnordost/streetcomplete/quests/clothing_bin_operator/AddClothingBinOperator.kt

```diff
@@ -43,6 +44,9 @@ class AddClothingBinOperator : OsmElementQuestType<String> {
 
     override fun getTitle(tags: Map<String, String>) = R.string.quest_clothes_container_operator_title
 
+    override fun getHighlightedElements(element: Element, getMapData: () -> MapDataWithGeometry) =
+        getMapData().filter("nodes with amenity = recycling and recycling_type = container")
```

Comment:
To stay consistent with the recycling quests:

```suggestion
        getMapData().filter("nodes with amenity = recycling")
```

I'm not sure about this though.

Reply:
🤷  ok